packages/now-next/src/index.ts

```diff
@@ -1034,6 +1050,10 @@ export const build = async ({
         continue: true,
       },
       { src: path.join('/', entryDirectory, '_next(?!/data(?:/|$))(?:/.*)?') },
+
+      // /_next/data routes for getServerProps pages
+      ...serverPropsRoutes,
```

Comment:
Are we sure this order is correct? It must come at least after filesystem AFAIK.

Reply:
Also, I feel like order is going to start mattering here that we have SSG and server props routes separate.

Reply:
Ah yeah, them being separate may be a problem since there might be a catch-all server props route that would override the SSG route or vice versa. Will look into grouping these together in the manifest so they are sorted properly together during the build so we don't have to attempt to sort them in the builder

Reply:
Opened PR here: https://github.com/zeit/next.js/pull/10622 replacing the `serverPropsRoutes` field with a `dataRoutes` field which combines and sorts these together to prevent conflicts

Reply:
Updated with changes from above PR